src/cmd/cli/command/track.go

```diff
@@ -23,11 +23,12 @@ func Track(name string, props ...P) {
 	if disableAnalytics {
 		return
 	}
+	var client cliClient.FabricClient = client
```

Comment:
This looks like it's unnecessary as it'll be cast when used in the go routine below.

Reply:
you're right, but was useful during dev